users/internal/handlers/users.go

```diff
@@ -86,3 +88,36 @@ func (u Users) GetOne(w http.ResponseWriter, r *http.Request) {
 		log.Error().Err(err).Msg("failed to encode response")
 	}
 }
+
+func (u Users) UpdateUserName(w http.ResponseWriter, r *http.Request) {
+	ctx := r.Context()
+	log := logger.FromContext(ctx)
+	id := chi.URLParam(r, "id")
+
+	var userData UserParams
+	if err := json.NewDecoder(r.Body).Decode(&userData); err != nil {
+		log.Error().Err(err).Msg("failed to parse params")
+		response.InternalError(w)
+		return
+	}
+
+	user := models.User{
+		ID:   id,
+		Name: userData.Name,
+	}
```

Comment:
looks like a part of business logic, the transport shouldn't know about models mapping